src/components/ViewServiceDetails/ViewServiceDetails.css

```diff
@@ -2,4 +2,8 @@
     .body{
         margin: 16px;
     }
+}
+
+.select-cmp {
+    width:100%;
```

Comment:
Watch out for spacing between width: and 100%

Reply:
added the format